Blog for Istio Community Meetup China (#10025)
istio/istio.io: content/en/blog/2021/istio-community-meetup-china/index.md

---
title: "The first local Istio meetup is here!"

> description: "Join Istio community experts in China"

Reviewer: add `.` in the end will fix the lint error:  
description: "Join Istio community experts in China."

> title: "The first local Istio meetup is here!"
> description: "Join Istio community experts in China."

Reviewer 2: ```suggestion
title: "Join us at the Istio Community Meetup in China"
description: "The Chinese Istio community comes together in Beijing."
```
Lots of places have run Istio meetups before; suggesting a more specific heading. (Not sure about the description yet)

(Will this be more useful when put on the ZH blog?)

publishdate: 2021-07-06
attribution: "Iris Ding (Intel), Maria Cruz (Google)"

> keywords: [community,meetup]

Reviewer 2: ```suggestion
keywords: [community,meetup,China]
```

---

> With the popularization of cloud native technology in China, Istio has also gained popularity in this corner of the world in the past two years. Almost all Chinese CSPs are creating and running their service mesh products based on Istio. Since the first IstioCon held in February 2021, the project has been widely welcomed by developers. Developers have also expressed an interest in participating in more Istio meetups and helping to grow the community at the local level. This is an opportunity to share and exchange across the globe.

Reviewer 2: ```suggestion
With the rapid popularization of cloud native technology in China, Istio has also gained popularity in this corner of the world. Almost all Chinese CSPs have creating and are running service mesh products based on Istio. 

We welcomed thousands of Istio users and developers to the first IstioCon in February 2021, and the attendees expressed an interest in participating in more meetups and helping to grow the community at the local level.
```

removed "This is an opportunity to share and exchange across the globe." as it's not really relevant in a China-only contex.t

> To this end, the Istio community united six partners from China Academy of Information and Communications Technology, Alibaba Cloud, Huawei Cloud, Intel, Tencent Cloud, and Tetrate to co-host the first domestic Istio Meetup China, and invited a number of industry experts to share comprehensive Istio technical practices with everyone. This will be an in-person meetup, we will serve some refreshments, and seats are limited, so we will operate on a first-come first-served basis.

Reviewer: Please register via [this link](https://www.huodongxing.com/event/7604616393700?td=3381727549788).

Author: added!

> To this end, the Istio community united six partners from China Academy of Information and Communications Technology, Alibaba Cloud, Huawei Cloud, Intel, Tencent Cloud, and Tetrate to co-host the first domestic Istio Meetup China, and invited a number of industry experts to share comprehensive Istio technical practices with everyone. This will be an in-person meetup, we will serve some refreshments, and seats are limited, so we will operate on a first-come first-served basis. Please register via [this link](https://www.huodongxing.com/event/7604616393700?td=3381727549788).

Reviewer 2: ```suggestion
To this end, the Istio community united six partners — the China Academy of Information and Communications Technology, Alibaba Cloud, Huawei Cloud, Intel, Tencent Cloud, and Tetrate — to co-host the first official Istio Community Meetup China. We have invited a number of industry experts to share comprehensive Istio technical practices with everyone at an in-person meetup. We will serve some refreshments, and seats are limited, so we will operate on a first-come first-served basis. [Please register to attend](https://www.huodongxing.com/event/7604616393700?td=3381727549788).
```

**Time and day:** 13:30-17:30 (CST), July 10, 2021

**Venue:** Industrial Internet Exhibition Hall, 2nd Floor, Research Building, China Academy of Information and Communications Technology, No. 52 Huayuan North Road, Haidian District, Beijing

### Agenda

> | Time (CST) | Session |
> | --- | --- |

Reviewer 2: ```suggestion
| Session time (CST) | Title |
| --- | --- |
```

longer title means dates don't wrap

| 13:30 - 13:50 | Sign in |
| 13:50 - 14:00 | Welcome speech-Craig Box (Istio steering committee member), Ding Shaojun (Iris Ding) (Intel cloud computing engineer)|
| 14:00 - 14:30 | Interpretation of the "Service Grid Technical Capability Requirements" Standard Yin Xia Mengxue, Engineer, Cloud Computing Department, Academy of Information and Communications Technology |

> | 14:30 - 15:00 | Service Grid Data Plane Hot Upgrade Technology Sharing Shi Zehuan Alibaba Cloud |

Reviewer: Service Grid  -> Service Mesh

Author: changed

> | 15:00 - 15:30 | "Envoy Principle Introduction and Online Problem Pit" Zhang Wei: Huawei Cloud Container Grid Data Plane Technical Expert |

Reviewer 3: s/Container Grid/service mesh

Author: changed

| 15:30 - 15:45 | Coffee break |
| 15:45 - 16:15 | Use eBPF to accelerate Istio/Envoy network - Zhong Luyao Intel |

> | 16:15 - 16:45 | Full-stack service grid-Aeraki helps you manage any seven-tier traffic in Istio Huabing Zhao Senior Engineer, Tencent Cloud |

Reviewer: service grid  -> service mesh

Author: changed

> | 13:30 - 13:50 | Sign in |
> | 13:50 - 14:00 | Welcome speech-Craig Box (Istio steering committee member), Ding Shaojun (Iris Ding) (Intel cloud computing engineer)|
> | 14:00 - 14:30 | Interpretation of the "Service Grid Technical Capability Requirements" Standard Yin Xia Mengxue, Engineer, Cloud Computing Department, Academy of Information and Communications Technology |
> | 14:30 - 15:00 | Service Mesh Data Plane Hot Upgrade Technology Sharing Shi Zehuan Alibaba Cloud |
> | 15:00 - 15:30 | "Envoy Principle Introduction and Online Problem Pit" Zhang Wei: Huawei Cloud Service Mesh Data Plane Technical Expert |
> | 15:30 - 15:45 | Coffee break |
> | 15:45 - 16:15 | Use eBPF to accelerate Istio/Envoy network - Zhong Luyao Intel |
> | 16:15 - 16:45 | Full-stack service mesh-Aeraki helps you manage any seven-tier traffic in Istio Huabing Zhao Senior Engineer, Tencent Cloud |
> | 16:45 - 17:15 | Securing workload deployment with Istio CNI Zhang Zhihan Tetrate |

Reviewer 2: ```suggestion
| 13:30 - 13:50 | Sign in |
| 13:50 - 14:00 | *Welcome*<br>Craig Box, Istio Steering Committee member, Google Cloud<br>Ding Shaojun (Iris Ding), cloud computing engineer, Intel|
| 14:00 - 14:30 | *Interpretation of the "Service Grid Technical Capability Requirements" Standard*<br>Yin Xia Mengxue, Engineer, Cloud Computing Department, Academy of Information and Communications Technology |
| 14:30 - 15:00 | *Service Mesh Data Plane Hot Upgrade*<br>Shi Zehuan, Alibaba Cloud |
| 15:00 - 15:30 | *Envoy Principle Introduction and Online Problem Pit*<br>Zhang Wei, Huawei Cloud Service Mesh Data Plane Technical Expert |
| 15:30 - 15:45 | Coffee break |
| 15:45 - 16:15 | *Use eBPF to accelerate Istio/Envoy network*<br>Zhong Luyao, Intel |
| 16:15 - 16:45 | *Full-stack service mesh: how Aeraki helps you manage any Layer 7 traffic in Istio<br>Huabing Zhao, Senior Engineer, Tencent Cloud |
| 16:45 - 17:15 | *Securing workload deployment with Istio CNI*<br>Zhang Zhihan, Tetrate |
```

> We want to thank our community members in China for working on producing this event, especially Iris Ding, Wei W Hu, and Jimmy Song. We hope many people can join this amazing event!

Reviewer: More names needs to added here: Zhonghu Xu, Xining Wang, Huabing Zhao, Thanks!

Author: added

> We want to thank our community members in China for working on producing this event, especially Iris Ding, Wei W Hu, Jimmy Song, Zhonghu Xu, Xining Wang, and Huabing Zhao. We hope many people can join this amazing event!

Reviewer 2: ```suggestion
We want to thank our community in China who have worked on this event: especially Iris Ding, Wei W Hu, Jimmy Song, Zhonghu Xu, Xining Wang, and Huabing Zhao. We hope you can join!
```

Nit: "we" is the authors, Maria and Iris, so saying "we want to thank Iris, Wei" etc might sound like patting yourself on the back if anyone read it too closely :)
